core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java

```diff
@@ -137,6 +152,24 @@ public void setCustomStrategies(String[] customStrategies) {
         return configItems;
     }
 
+    public List<StrategyConfiguration> strategyConfigurations(String[] strategies) {
```

Comment:
Is it possible to incorporate this method to the `load()` method - as another option? To be possible to call the Configuration load method only once in other integration cases (such as Che)

Reply:
Also, I would propose changing the method name to `loadStrategyConfigurations` to express what it really does

Reply:
> Is it possible to incorporate this method to the load() method - as another option? To be possible to call the Configuration load method only once in other integration cases (such as Che)

if we move this method in `load()`, then we won't have strategy dependency added on classpath in case of mvn extension. So we won't get any any implementation of `strategyConfiguration`

Reply:
I meant to have a possibility to call `load(String[] strategies)` to load everything at once... both the current `load()` and `strategyConfigurations(String[] strategies)` would stay